_dev/src/components/product-feed-page/submitted-products/products-status-card.vue

```diff
@@ -35,17 +39,21 @@
 
 <script>
 import ProductsStatusType from '@/enums/product-feed/products-status-type';
+import {
+  SyncHystoryType,
+} from '@/enums/product-feed/sync-history.ts';
 
 export default {
   props: {
     productStatus: {
       type: Object,
       required: true,
       validator(obj) {
+        console.log(`obj.statusOfProducts: ${obj.statusOfProducts}`, `obj.numberOfProducts: ${obj.numberOfProducts}`);
```

Comment:
to remove

Reply:
Still a WIP and a draft, it will be removed.